### PR TITLE
[ESS][8.14] ES|QL rule is GA'ing 

### PR DESCRIPTION
**Pull request by vitaliidm**
- addresses https://github.com/elastic/kibana/issues/178422 and contributes to contributes to https://github.com/elastic/security-docs/issues/5149
- relevant PR in Kibana: https://github.com/elastic/kibana/pull/179855

Removed the tech preview label that was applied to UI and API docs for the ES|QL rule:
- [About detection rules | Rule types](https://security-docs_bk_5032.docs-preview.app.elstc.co/guide/en/security/master/about-rules.html#rule-types)
- [Create a detection rule | Create an ES|QL rule](https://security-docs_bk_5032.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-esql-rule)
- [Create rule API](https://security-docs_bk_5032.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#req-fields-query-threshold)

### CODE REVIEW

**Comment by vitaliidm** [comment]
Kibana PR has been merged: https://github.com/elastic/kibana/pull/179855

File: docs/events/timeline-ui-overview.asciidoc [hunk 1]
```diff
@@ -196,8 +196,6 @@ From the *Correlation* tab, you can also do the following:
 [[esql-in-timeline]]
 == Use {esql} to investigate events 
 
-preview::["Do not use {esql} on production environments. This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
-
 NOTE: The {esql} tab is available by default. Since it's in technical preview, you can remove it by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
```
Comment (nastasha-solomon) [inline docs/events/timeline-ui-overview.asciidoc:199]: @michaelolo24 is the feature flag that enables/disables the **ES|QL** Timeline being removed in 8.14? In other words, will the **ES|QL** Timeline tab be available by default when users upgrade to 8.14?

cc: @paulewing

Comment (nastasha-solomon) [reply docs/events/timeline-ui-overview.asciidoc:199]: Verified with both via Slack that it's ok to remove the note on line 199.

Comment (nastasha-solomon) [reply docs/events/timeline-ui-overview.asciidoc]: Addressed in [69c8aaa](https://github.com/elastic/security-docs/pull/5032/commits/69c8aaa68a48fb97db6e879de1019f47befa8209)

**Review by nastasha-solomon — APPROVED** [review]
I checked the preview and everything looks good!

**Comment by michaelolo24** [comment]
> Updating this comment:
> 
> * Tech preview label removed from all ES|QL feature docs ✅
> * Note about enabling/disabling the **ES|QL** Timeline tab has been removed ✅
> 
> @michaelolo24 the only thing that's left is for me to refresh the screenshot of the **ES|QL** tab in Timeline. Right now, it has the tech preview icon and the banner notifying users that ES|QL is still in technical preview.
> 
> esql-tab

Will get you an updated screenshot asap!

**Comment by michaelolo24** [comment]
Thanks for making the changes @vitaliidm ! The PR removing the beta tag from the tab is here:

https://github.com/elastic/kibana/pull/180838

**Review by michaelolo24 — APPROVED** [review]

**Comment by nastasha-solomon** [comment]
Moving this PR back to the draft state as I make additional updates to screenshots and AI assistant docs.

**Comment by nastasha-solomon** [comment]
@michaelolo24 heads up that I'm moving the Timeline-related changes over to https://github.com/elastic/security-docs/pull/5139.

**Review by nastasha-solomon — APPROVED** [review]

**Review by benironside — APPROVED** [review]
:shipit:

**Pull request merged**